README.md

```diff
@@ -13,7 +13,7 @@ Run `build_and_run.sh`. This will install Tulsi.app inside `$HOME/Applications`
 
 * `-b`: Bazel binary that Tulsi should use to build and install the app (Default is `bazel`)
 * `-d`: The folder where to install the Tulsi app into (Default is `$HOME/Applications`)
-* `-x`: The Xcode version Tulsi should be built for (Default is `11.5`)
+* `-x`: The Xcode version Tulsi should be built for (Default is `11.7`)
```

Comment:
Obviously, I do realize there's this option to allow users to build Tulsi for an Xcode version different from 11.5. 

I thought it might be nice to stay onto the latest version of Xcode. It lowers the barrier to entry.

An alternative if you don't want to get the default value of of sync with what is used in CI right now, might be to make the error message clearer, suggesting to use the `-x` option.